src/components/NavBar/ProfileButton/index.tsx

```diff
@@ -3,15 +3,16 @@ import { useNavigate } from 'react-router-dom';
 import { useSelector } from 'react-redux';
 import clsx from 'clsx';
 import ProfileImage from 'assets/images/profile/user-image.png';
-import styles from './index.module.scss';
 import { RootState } from 'reducers';
-
+import styles from './index.module.scss';
 
 const ProfileButton: FC = () => {
-
   const { user } = useSelector((state: RootState) => state.user);
+  const profile = useSelector((state: RootState) => state.profile.profile);
   const navigate = useNavigate();
-
+  const pictureFunc = () => {
+    return profile?.picture ?? ProfileImage;
```

Comment:
I forgot about this thing. Are you saving picture as empty string at any point of time? Then this `??` won't work, I think need to use `||`. You can test which one returns `falsy` for left side in all null, undefined and empty string